vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -36,8 +47,7 @@ export default Vue.extend({
           sessionStorage.removeItem('KEYCLOAK_REFRESH_TOKEN')
           sessionStorage.removeItem('KEYCLOAK_TOKEN')
           sessionStorage.removeItem('REGISTRIES_TRACE_ID')
-          sessionStorage.removeItem('AUTH_WEB')
-          window.location.assign('/auth')
+          window.location.assign('/')
```

Comment:
What is this change for? I guess that /auth was added to take the user to the login page, if it's '/' that may have additional redirection.

Reply:
Whoa. didn't catch that. Fixing here now.